timemachine/datasets/water_exchange/README.md

```diff
@@ -0,0 +1,5 @@
+This folder contains a list of prepared water structures for use in the examples/water_sampling.py script:
+
+bb_centered_neutral.sdf C90-buckyball with zero charges (very difficult to insert any water inside)
+bb_centered_espaloma.sdf C90-buckyball with espaloma charges (inserts 1-2 waters inside)
```

Comment:
nit: which version?

Reply:
I don't remember, I emailed @yuanqing-wang the sdf file - not sure it matters too much